registration/tests/serializers/test_field_list.py

```diff
@@ -56,6 +64,7 @@ def test_field_list_serializer(self):
                     ],
                     "child_fields": [FieldSerializer(self.child_field).data],
                     "guest_fields": [FieldSerializer(self.guest_field).data],
+                    "session_fields": [FieldSerializer(self.session_field).data],
```

Comment:
i think this test is failing because we also need to add session_fields to the test below - hopefully that should fix it!